packages/playground/src/weblets/tf_kubernetes.vue

```diff
@@ -131,7 +131,7 @@ async function deploy() {
     layout.value.setStatus("success", "Successfully deployed a Kubernetes cluster.");
     layout.value.openDialog(k8s, deploymentListEnvironments.k8s);
   } catch (e) {
-    layout.value.setStatus("failed", normalizeError(e, "Failed to deploy kubernetes cluster."));
+    layout.value.setStatus("failed", normalizeError(e, "Failed to deploy Kubernetes cluster."));
```

Comment:
a Kubernetes cluster.

Reply:
Yes! Was fixed in the 2nd commit. thanks and good catch!

Reply:
It's not fixed yet.